dxr/plugins/descriptor/__init__.py

```diff
@@ -119,9 +119,9 @@ def generic_describe(self):
         try:
             description_re = re.compile(r'(?:{}|{}|description)\
```

Comment:
I don't think you can just throw line breaks and other whitespace into the middle of regexes and have it ignored, unless you set the re.X flag.